test.sh

```diff
@@ -1,35 +1,66 @@
 #!/usr/bin/env bash
 
 function finish {
+  echo 'Removing test environment'
+  echo '-----'
   docker-compose down -v
 }
 trap finish EXIT
 
-echo "Running tests"
+function main() {
+  prepareOutputDir
+  createTestEnvironment
+  loadTestPolicy
+  runTests
+}
+
+function prepareOutputDir() {
+  # Clean then generate output folder locally
+  rm -rf output
+  mkdir -p output
+}
 
-# Clean then generate output folder locally
-rm -rf output
-mkdir -p output
+function createTestEnvironment() {
+  echo "Creating test environment"
+  echo '-----'
+
+  # Build test container & start the cluster
+  # docker-compose pull postgres possum
```

Comment:
I took care of this, we're green on master now. Sorry about that Oren!

https://jenkins.conjur.net/blue/organizations/jenkins/api-java/detail/master/6/changes